modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/product/navigation/control/menu/EditFileEntryHeaderProductNavigationControlMenuEntry.java

```diff
@@ -74,8 +73,17 @@ public boolean isShow(HttpServletRequest httpServletRequest)
 		PortletDisplay portletDisplay = themeDisplay.getPortletDisplay();
 
 		if ((portletDisplay == null) ||
-			!Objects.equals(
-				portletDisplay.getId(), DLPortletKeys.DOCUMENT_LIBRARY_ADMIN)) {
+			(!StringUtil.equals(
+				"/document_library/edit_file_entry",
+				ParamUtil.getString(
+					httpServletRequest,
+					portletDisplay.getNamespace() + "mvcRenderCommandName")) &&
```

Comment:
A very minor thing: isn't there an extra set of parenthesis here? It is weird that the auto SF didn't catch this.

Reply:
Hi @adolfopa, thanks for reviewing the pr. This set (!StringUtil actually added by the gw formatSource automatically, or  an other one is the extra? Thx

Reply:
Sorry, you're right. Everything is fine; sorry!